src/main/java/com/amazon/sqs/javamessaging/AmazonSQSMessagingClientWrapper.java

```diff
@@ -101,48 +103,9 @@ public AmazonSQSMessagingClientWrapper(AmazonSQS amazonSQSClient, AWSCredentials
      * <code>JMSException</code>.
      * @return amazonSQSClient
      */
-    public AmazonSQS getAmazonSQSClient() {
+    public SqsClient getAmazonSQSClient() {
         return amazonSQSClient;
     }
-
-    /**
-     * Sets SQS endpoint and wraps IllegalArgumentException. 
-     * Deprecated. Instead of manipulating settings of existing AmazonSQS client, provide correct configuration when creating it through SQSConnectionFactory constructors.
-     * 
-     * @param endpoint
-     *            The endpoint (ex: "sqs.us-east-1.amazonaws.com") of the region
-     *            specific AWS endpoint this client will communicate with.
-     * @throws JMSException
-     */
-    @Deprecated
-    public void setEndpoint(String endpoint) throws JMSException {
-        try {
-            amazonSQSClient.setEndpoint(endpoint);
-        } catch (IllegalArgumentException ase) {
-            JMSException jmsException = new JMSException(ase.getMessage());
-            throw (JMSException) jmsException.initCause(ase);
-        }
-    }
-    
-    /**
-     * Sets SQS region and wraps <code>IllegalArgumentException</code>. 
-     * Deprecated. Instead of manipulating settings of existing AmazonSQS client, provide correct configuration when creating it through SQSConnectionFactory constructors.
-     * 
-     * @param region
-     *            The region this client will communicate with. See
-     *            {@link Region#getRegion(com.amazonaws.regions.Regions)} for
-     *            accessing a given region.
-     * @throws JMSException
-     */
-    @Deprecated
-    public void setRegion(Region region) throws JMSException {
-        try {
-            amazonSQSClient.setRegion(region);
-        } catch (IllegalArgumentException ase) {
-            JMSException jmsException = new JMSException(ase.getMessage());
-            throw (JMSException) jmsException.initCause(ase);
-        }
-    }
     
     /**
      * Calls <code>deleteMessage</code> and wraps <code>AmazonClientException</code>. This is used to
```

Comment:
I see in other comments you updated the documented exception type to match the catch block. Should we do the same here?

Reply:
Good catch! Will update this one and scan the entire package again.